client/src/components/CheckboxContainer.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const CheckboxContainer = (props) => {
+    let mappedItems = [];
+    if (props.frequentItems !== undefined && props.frequentItems.length > 0) {
+        mappedItems = props.frequentItems.map((element, i) => {
+            if (i === 0) {
+                return <div key={i}><input type="checkbox" value="test" checked={element.checked} onChange={() => {props.checkHandler(element.id)}}/><label className={element.checked ? "checkbox-label" : "grayed-out"}>This item: {element.itemName} ${element.price}</label></div> 
+            } else {
+                return <div key={i}><input type="checkbox" value="test" checked={element.checked} onChange={() => {props.checkHandler(element.id)}}/><label className={element.checked ? "" : "grayed-out"}><a href="#" className={element.checked ? "" : "grayed-out"} onClick={()=>{props.itemClick(element.id)}} >{element.itemName}</a>  ${element.price}</label></div>
```

Comment:
I think that for better readability, keeping lines < 120 characters is what's usually the industry standard (sometimes even < 80 I think). Next time, this would be better for us when we're reviewing your code to understand what's happening. **Line 10** is ~347 characters long, lol.